roles/docker/docker-storage/defaults/main.yml

```diff
@@ -3,6 +3,9 @@ docker_container_storage_setup_version: v0.6.0
 docker_container_storage_setup_profile_name: kubespray
 docker_container_storage_setup_storage_driver: devicemapper
 docker_container_storage_setup_container_thinpool: docker-pool
+docker_container_storage_setup_vg: docker
```

Comment:
In the template, this vg is optional:
```
{%if docker_container_storage_setup_vg is defined%}VG={{docker_container_storage_setup_vg}}{%endif%}
```
Is it in fact not optional?

Reply:
this vg is optional, I delete  it.